cou/zaza_utils/generic.py

```diff
@@ -66,25 +66,26 @@ def get_network_config(net_topology, ignore_env_vars=False, net_topology_file="n
     return net_info
 
 
-def get_pkg_version(application, pkg):
+def get_pkg_version(application_charm, pkg, model_name=None):
```

Comment:
As Erhan mention we should not touch these function, since they are copied from zaza.

Reply:
why are we copying files from zaza? can the module not be imported and used?